onnxruntime/core/session/inference_session.cc

```diff
@@ -556,7 +556,11 @@ common::Status InferenceSession::Load(const std::string& model_uri) {
 
   if ((has_explicit_type && model_type == "ORT") ||
       (!has_explicit_type && inference_session_utils::IsOrtFormatModel(model_uri))) {
+#if defined(ENABLE_ORT_FORMAT_LOAD)
     return LoadOrtModel(model_uri);
+#else
+    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "ORT format model is not supported in this build.");
```

Comment:
>this build [](start = 96, length = 10)

nit: instead of 'this build', would it be nicer to have:

"is not supported when ENABLE_ORT_FORMAT_LOAD is off"?

Reply:
Yeah. Depends how you build as to what disabled it. Could be a flag to build.py (called through build.bat or build.sh). Could be direct use of cmake.